server/parameters/config.rs

```diff
@@ -18,14 +18,14 @@ pub struct Config {
 impl Config {
     pub fn new() -> Self {
         Self {
-            server: ServerConfig { address: SocketAddr::from_str("localhost:1729").unwrap() },
-            storage: StorageConfig { data: "runtimedata/server/data".into() },
+            server: ServerConfig { address: SocketAddr::from_str("127.0.0.1:1729").unwrap() },
+            storage: StorageConfig { data: "runtimedata/server/data".into() }, // TODO: Settle this path and update defaults in assembly rules
```

Comment:
If we decide on the path, we can encode it into the docker assembly.

Reply:
Was it separately encoded in the sources and in the docker assembly area before?